modules/light-clients/08-wasm/keeper/snapshotter.go

```diff
@@ -110,12 +111,12 @@ func restoreV1(_ sdk.Context, k *Keeper, compressedCode []byte) error {
 		return errorsmod.Wrap(err, "failed to uncompress wasm code")
 	}
 
-	checksum, err := k.wasmVM.StoreCodeUnchecked(wasmCode)
+	checksum, err := ibcwasm.GetVM().StoreCode(wasmCode)
```

Comment:
`StoreCodeUnchecked` should be used here!

Reply:
https://github.com/cosmos/ibc-go/pull/5248/commits/381ad38ab60dc5c9fc6ee72d900e3a843ff4b2a6